packages/storage/src/cookies.ts

```diff
@@ -63,9 +63,10 @@ const setCookie = isServer
       return;
     }
     const serializedOptions = serializeCookieOptions(options);
-    responseHeaders.append(
+    const cleanedCookies = responseHeaders.get("Set-Cookie").replace(new RegExp(`(^|, )${key}=[^,]+,?`, "g"), "$1");
```

Comment:
why replacing with`$1` and not with an emtpy string?

Reply:
Because otherwise, I would potentially remove too many commas; but maybe, I'll just remove the `,?` and then use `""`.

Reply:
yeah do that, it would be a bit weird if in the response header randomly a `$1` appears, because as far as i can see this is never replaced against anything different

Reply:
$1 will be replaced with the first capture, in this case an empty string or ", ".

Reply:
Ahh ok, didn't know that. If it is so than everything should be fine

Reply:
It was still wasting CPU cycles, so I've fixed it.